controllers/usersignup/usersignup_controller_test.go

```diff
@@ -353,7 +353,7 @@ func TestUserSignupWithAutoApprovalWithoutTargetCluster(t *testing.T) {
 	AssertMetricsCounterEquals(t, 0, metrics.UserSignupDeactivatedTotal)
 	AssertMetricsCounterEquals(t, 1, metrics.UserSignupApprovedTotal)
 	AssertMetricsCounterEquals(t, 1, metrics.UserSignupUniqueTotal)
-	segmenttest.AssertMessageQueued(t, r.SegmentClient, userSignup, segment.AccountActivated)
+	segmenttest.AssertMessageQueuedForProvisionedMur(t, r.SegmentClient, userSignup, murtest.AssertThatMasterUserRecord(t, userSignup.Name, r.Client).Get())
```

Comment:
in this case, it's enough to move it a few lines below because at line 358 there is the assertion of the MUR

Reply:
Done in https://github.com/codeready-toolchain/host-operator/pull/787/commits/a60a2d65c75591ff5fa145824a2bb1e3911e06c7